teamManager/teamManager.py

```diff
@@ -1155,6 +1156,7 @@ def get_current_franchise_role(self, user: discord.Member):
                     return role
             except:
                 continue
+        return None
```

Comment:
why explicit `None` return?

Reply:
I thought `None` was returned by default, but it isn't. This made things a little more confusing when I was debugging my issues. When I was looking at what a value was, it wasn't showing anything, whereas when I return `None` explicitly, I could see that my value wasn't being properly defined.

Why that was happening is a different question. Answer? I don't know lol.

Reply:
Actually I remember what my issue was, I was trying to get franchise/team info after I removed roles (🤦‍♂️). I updated these to explicitly return `None` to help find out what was happening.